engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/plan/GraphQLQueryPlan.kt

```diff
@@ -0,0 +1,15 @@
+package graphql.nadel.enginekt.plan
+
+import graphql.nadel.Service
+import graphql.nadel.enginekt.transform.result.GraphQLResultTransform
+import graphql.normalized.NormalizedField
+
+data class GraphQLQueryPlan(
```

Comment:
we need to rename this: it is not a query plan but a Result plan imho, if it is only concerned with results.